docs/reference/autoscaling/index.asciidoc

```diff
@@ -1,10 +1,9 @@
 [role="xpack"]
-[testenv="platinum"]
+[testenv="enterprise"]
 [[xpack-autoscaling]]
-[chapter]
```

Comment:
My asciidoc skills failed me here - removing this made this work, otherwise the sub page would appear on the same level in the navigation.